lib/agris/api/grain/new_ticket_application.rb

```diff
@@ -27,7 +27,7 @@ class NewTicketApplication
         def initialize(hash = {})
           super
 
-          @record_type = 'GRNT1'
+          @record_type = "GRNT1"
```

Comment:
Under our rules these should be single quotes. I think something went wrong with the rubocop config.

Reply:
This file should be right https://github.com/westernmilling/otto/blob/master/.rubocop.yml though it could probably be shortened as I think part of those are defaults now.